src/components/game/game.jsx

```diff
@@ -63,60 +26,50 @@ class Game extends Component {
 
     let status;
     if (winner) {
-      status = `${winner} is the Winner!`;
+      status = `${winner.winner} is the Winner!`;
```

Comment:
This prop seems confusing, would winner.name not be better?

Reply:
Will change it